server/job.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/pluginapi/cluster"
+	"github.com/pkg/errors"
+)
+
+const (
+	jobKeyStartSurveyJob = "job_start_survey"
+
+	// TODO - update this to 15 minutes once ready for production
+	startSurveyJonInterval = 15 * time.Second
+)
+
+func (p *Plugin) startScheduledJobs() error {
+	if err := p.startSurveyStartJob(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *Plugin) startSurveyStartJob() error {
```

Comment:
It is intended that this name has `start` twice?

Reply:
ah, my bad. Renamed to `JobManageSurveyStatus` now.